flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/debezium/reader/BinlogSplitReaderTest.java

```diff
@@ -324,7 +317,93 @@ public void testReadBinlogFromLatestOffset() throws Exception {
                 };
         List<String> actual =
                 readBinlogSplitsFromLatestOffset(dataType, configuration, expected.length);
-        assertEquals(Arrays.stream(expected).sorted().collect(Collectors.toList()), actual);
+        assertThat(actual, containsInAnyOrder(expected));
+    }
+
+    @Test
+    public void testReadBinlogCrossTransaction() throws Exception {
+        customerDatabase.createAndInitialize();
```

Comment:
I think we can remove this test now, because it's covered by `MySqlSourceReaderTest`?